MqttClient_Hello/app/application.cpp

```diff
@@ -11,12 +11,25 @@
 void startMqttClient();
 void onMessageReceived(String topic, String message);
 
-Timer procTimer;
+Timer procTimer, mqttTimer;
```

Comment:
Can you not use the same timer? Would they ever both be needed at the same time?